webrtc-stats.html

```diff
@@ -2751,7 +2778,7 @@ <h3>
         </h3>
         <div>
           <pre class="idl">dictionary RTCAudioHandlerStats : RTCMediaHandlerStats {
-             boolean             voiceActivityFlag;
+              boolean              voiceActivityFlag;
```

Comment:
Do we keep this here or to the RTP stats...

Reply:
Let's move it to RTP. In that case it should definitely be in RTCInboundRtpStreamStats (reflecting the getSSRCs() which are for receivers), but I question whether it should be in RTCOutboundRtpStreamStats (there is no getSSRCs() on the senders). In this PR, to not introduce any changes other than moving, I moved it to both,  but we nobody has ever implemented sender-side voiceActivityFlag. (Although we haven't implemented the VAD in getSSRCs() either so probably feature at risk regardless)